test/framework/bootstrap/kind_util.go

```diff
@@ -147,7 +148,18 @@ func loadImage(ctx context.Context, cluster, image string) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to access container runtime")
 	}
-
+	// in the nominal E2E scenario images have been locally built and added to cache
+	exists, err := containerRuntime.ImageExistsLocally(ctx, image)
+	if err != nil {
+		return errors.Wrapf(err, "error listing local image %q", image)
```

Comment:
String does not really match, maybe we even don't need error wrapping here?
```suggestion
		return errors.Wrap(err, "error checking if image exists locally")
```

Reply:
I think it's worth printing out the image, changed to `%s`